src/utils/getIdentifierInputDisplay.tsx

```diff
@@ -0,0 +1,31 @@
+import { IdentifierInput } from "../types";
+
+const processIdentifierInputValue = (
+  identifierInput: IdentifierInput
+): string => {
+  return identifierInput.textInputType === "PHONE_NUMBER"
+    ? `****${identifierInput.value.slice(identifierInput.value.length - 4)}`
```

Comment:
Perhaps can use `padStart`?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart

Reply:
From discussion with Pat, we can display fixed 4 *s at the start. Let me ask if she prefers to pad as per original length, if yes I'll change to this. If not, I think might be more straightforward to do 4 *s?

Reply:
Oh I was thinking to move this to some generic `maskText` function that can be used for anything, including NRIC, so just need to configure how many characters to show.

Reply:
Pat said prefers to retain length, so I changed to padStart. Thanks for the suggestion 🙌